pkgs/top-level/all-packages.nix

```
@@ -29678,7 +29678,7 @@ with pkgs;
 
   edlin = callPackage ../applications/editors/edlin { };
 
-  o = callPackage ../applications/editors/o {
+  orbiton = callPackage ../applications/editors/orbiton {
     buildGoModule = buildGo119Module; # go 1.20 build failure
   };
```

```suggestion
  orbiton = callPackage ../applications/editors/orbiton { };
```

Builds with go 1.20 (since https://github.com/xyproto/orbiton/releases/tag/v2.59.3, I think)

---

Can you add an alias in https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/aliases.nix as well? :)

---

Fixed